commit a684ec9ea628e057e26aa91653eb94877e3ff8f8 (refs/changes/79/4779/2)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2018-09-02T13:36:54-07:00 (1 year, 1 month ago)
    
    OS-7211 root should recognize pkgsrc in the gz
    Reviewed by: Jorge Schrauwen <sjorge@blackdot.be>

diff --git a/overlay/generic/root/.profile b/overlay/generic/root/.profile
index 80e557e4..906d43ba 100644
--- a/overlay/generic/root/.profile
+++ b/overlay/generic/root/.profile
@@ -5,10 +5,10 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
-PATH=/usr/bin:/usr/sbin:/smartdc/bin:/opt/smartdc/bin:/opt/local/bin:/opt/local/sbin:/opt/smartdc/agents/bin
-MANPATH=/usr/share/man:/smartdc/man:/opt/smartdc/man:/opt/local/man
+PATH=/usr/bin:/usr/sbin:/smartdc/bin:/opt/smartdc/bin:/opt/local/bin:/opt/local/sbin:/opt/tools/bin:/opt/tools/sbin:/opt/smartdc/agents/bin
+MANPATH=/usr/share/man:/smartdc/man:/opt/smartdc/man:/opt/local/man:/opt/tools/man
 PAGER=less
 export PATH MANPATH PAGER
